lib/cartman/cart.rb

```diff
@@ -4,9 +4,9 @@ class Cart
 
     attr_reader :discounted
 
-    def initialize(uid, discounted = false)
+    def initialize(uid, discounted = '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.